### DISCUSSION TITLE
Sourcemaps on already minified react code

### DISCUSSION THREAD

**Discussion by onxb** [javascript-obfuscator/javascript-obfuscator#1216] [Q&A]
Hello, I am looking to daisy chain two source maps!  obfuscated -> minified js -> typescript
When I obfuscate a create-react-app minified  .js file and request a source-map, the resulting map only takes me as far as the minified .js. I want the typescript as the source.

Do you know of any processes, builders, etc that could provide an obfuscated .js with a map back to the original typescript?

**Comment by Bradeep** [comment]
Hey! Did you got any solution for this?
Struggling to get the source-code back from the obfuscated code
